docs/examples/LayoutDisplayGrid/AutoLayoutField.js

```diff
@@ -0,0 +1,11 @@
+import React from 'react';
```

Comment:
Please put all these files into one more folder to signalize that those are bound to the `examples-more` (`examples-random`) section.